client/src/controllers/LinkController.ts

```diff
@@ -0,0 +1,68 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class LinkController extends Controller<HTMLElement> {
+  static values = {
+    attrName: { default: 'href', type: String },
+    preserveKeys: { default: [], type: Array },
+    reflectKeys: { default: ['__all__'], type: Array },
+  };
```

Comment:
This shows the limits of one-word Stimulus controllers to me. I think we should add a docs comment and one-line comments for all the options to be clearer about what they do. Something to revisit once the 6.0 release candidate is out of the door.